addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1240,6 +1240,54 @@ registry.ImagesLazyLoading = publicWidget.Widget.extend({
     },
 });
 
+registry.BackgroundShape = publicWidget.Widget.extend({
+    selector: '.o_we_shape',
+
+    /**
+     * @override
+     */
+    start() {
+        this._onBackgroundShapeResize();
+        window.addEventListener('resize', this._onBackgroundShapeResize.bind(this));
+        return this._super(...arguments);
+    },
+    /**
+     * @override
+     */
+    destroy() {
+        window.removeEventListener('resize', this._onBackgroundShapeResize);
+        this._super(...arguments);
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    _onBackgroundShapeResize() {
+        // Due to a bug on Chrome when using browser zoom, there is sometimes
+        // a gap between sections with shapes. This gap is due to a rounding
+        // issue when positioning the SVG background image. This code fixes
+        // that by ensuring that shape elements always have a width value as
+        // close to an integer as possible.
+        this.target.style.left = '';
+        this.target.style.right = '';
+        // Get the decimal part of the shape element width.
+        let decimalPart = this.target.getBoundingClientRect().width % 1;
+        // Round to two decimal places.
+        decimalPart = Math.round((decimalPart + Number.EPSILON) * 100) / 100;
+        // If there is a decimal part. (e.g. Chrome + browser zoom enabled)
+        if (decimalPart > 0) {
+            // Compensate for the gap by giving an integer width value to the
+            // shape by changing its "right" and "left" positions.
+            const offset = ((decimalPart < 0.5 ? decimalPart : decimalPart - 1) / 2);
+            this.target.style.left = this.target.style.right = offset + 'px';
+        }
```

Comment:
Doing this, as an example:
- I have a shape of 305.74px width
- You take the decimal part: 0.74px
- It is bigger than 0.5, so you do, x - 1 / 2 -> -0.13px
- You apply -0.13px to left and right (so you make it arguably bigger)

1. Are you sure an horizontal scrollbar never appears?
2. I am still confused this fix being about the width only. Doing this, you effectively reduce Chrome error I guess, but can't you do the same to the height after having changed the width to reduce the error even further? Maybe getting rid of the 0.02px you still get sometimes?

Reply:
1 => According to my tests, the scrollbar never appears before an overflow of less than 0.55 px. so I guess it's safe as this code allow a max overflow of 0.5 px.
2 => No, even if I change the height to get an integer height value, the issue with the 0.02px is still there. 
I think the issue is related to the width because the SVG must be set to 100% of the width and it is based on the rounded value (integer) and not the value with the decimal. So the SVG width is an integer when the container width is not.

Reply:
> 1 => According to my tests, the scrollbar never appears before an overflow greater than 0.333 px. so I guess it's safe as this code allow a max overflow (to the right) of 0.25 px. 2 => No, even if I change the height to get an integer height value, the issue with the 0.02px is still there. I think the issue is related to the width because the SVG must be set to 100% of the width and it is based on the rounded value (integer) and not the value with the decimal. So the SVG width is an integer when the container width is not.
(I added an extra 0.015px offset and it prevents the remaining line from appearing..)

Reply:
@qsm-odoo I made the changes. Let's hope we get rid of this bug thanks to this ugly fix